### PR TITLE
Alert that 2FA must be authorized in app/config/parameters.yml

### PR DESCRIPTION
**Pull request by nicofrand**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | no
| License       | MIT

Alert that 2FA must be authorized in app/config/parameters.yml

### CODE REVIEW

**Comment by nicosomb** [comment]
Can you add this chapter on the other translations please? 
We'll translate it in :gb:, :de: (ping @jlnostr @Strubbl) and :it: (ping @matteocoder).

**Comment by matteocoder** [comment]
Got it! 👍

**Comment by nicofrand** [comment]
Should be OK now.

File: docs/de/user/configuration.rst [hunk 1]
```diff
@@ -66,7 +66,9 @@ Zwei-Faktor-Authentifizierung (2FA)
     Die Zwei-Faktor-Authentifizierung (2FA) dient dem Identitätsnachweis eines Nutzers mittels der
     Kombination zweier verschiedener und insbesondere unabhängiger Komponenten (Faktoren).
 
-https://de.wikipedia.org/wiki/Zwei-Faktor-Authentifizierung
+    https://de.wikipedia.org/wiki/Zwei-Faktor-Authentifizierung
+    
+**Warning**: enabling 2FA from the configuration interface is only possible is it has been authorized before in `app/config/parameters.yml` by setting the *twofactor_auth* property to `true` (do not forger to empty `/var/cache` after modification).
```
Comment (jlnostr) [inline docs/de/user/configuration.rst]: I am a bit lazy to clone the repo for this small translation, so I post it below:
```
**Warnung:** Das Aktivieren von 2FA über das Konfigurations-Interface ist nur möglich, wenn vorher in der `app/config/parameters.yml` die `twofactor_auth`-Eigenschaft auf `true` gesetzt wurde (nach der Konfiguration das Leeren des Cache unter `/var/cache` nicht vergessen).
```

**Comment by nicofrand** [comment]
@matteocoder If you paste your translation here I'll integrate it directly into this PR, if you want.

File: docs/en/user/configuration.rst [hunk 2]
```diff
@@ -68,7 +68,9 @@ Two factor authentication (2FA)
     Two-factor authentication (also known as 2FA) is a technology patented in 1984
     that provides identification of users by means of the combination of two different components.
 
-https://en.wikipedia.org/wiki/Two-factor_authentication
+    https://en.wikipedia.org/wiki/Two-factor_authentication
+    
+**Warning**: enabling 2FA from the configuration interface is only possible if it has been authorized before in `app/config/parameters.yml` by setting the *twofactor_auth* property to `true` (do not forget to empty `/var/cache` after modification).
```
Comment (j0k3r) [inline docs/en/user/configuration.rst]: Could you please avoid saying to clear the folder?
Instead it's more safe to use `php bin/console cache:clear -e=prod`

**Comment by matteocoder** [comment]
@nicofrand Really? Do I have to paste it as a comment?

**Comment by nicofrand** [comment]
Ok @j0k3r.

Yep @matteocoder, if you want to, as long as this PR is not merged I can include your work. (Please mind the previous comment from j0k3r).

Le 19 févr. 2017 10:20 PM, matteocoder <notifications@github.com> a écrit :

@nicofrand Really? Do I have to paste it as a comment?

**Comment by matteocoder** [comment]
**Attenzione**: abilitare la 2FA dall'interfaccia di configurazione è possibile solamente se ciò è stato abilitato precedentemente in `app/config/parameters.yml` impostando la proprietà *twofactor_auth* su `true` (non dimenticate di eseguire il comando ``php bin/console cache:clear -e=prod`` dopo la modifica).

If this is ok I can close my separate PR

**Comment by nicofrand** [comment]
This PR now includes all languages (I hope I did not break the german translation though).
Sorry @matteocoder, I did not see your PR :s.
You can close it indeed.

File: docs/it/user/configuration.rst [hunk 3]
```diff
@@ -47,9 +47,11 @@ Potete cambiare il vostro nome, il vostro indirizzo email e abilitare l'``Autent
 Autenticazione a due fattori (2FA)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli utenti tramite una combinazione di due componenti differenti.
+    L'autenticazione a due fattori (conosciuta anche come 2FA) é una tecnologia brevettata nel 1984 che offre l'identificazione degli utenti tramite una combinazione di due componenti differenti.
```
Comment (nicosomb) [inline docs/it/user/configuration.rst]: Please remove this useless tab.

Comment (nicofrand) [inline docs/it/user/configuration.rst]: not useless, look at the preview, this is used to create some quote block.

**Pull request merged**